css/style.css

```diff
@@ -16,4 +12,259 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* General styles */
+
+:root {
+    
+    --grey: #616668;
+}
+
+html, body {
+    font-family: 'Roboto', sans-serif;
+    -webkit-font-smoothing: antialiased;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    width: 100%;    
+    font-family: Roboto, sans-serif; 
+    overflow-x: hidden;  
+}
+
+main {   
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding-top: 20px;
+}
+
+/* img{
+    max-width: 100%;
+    height: auto;
+    display: block;
+} */
+
+/* Navigation */
+
+nav {
+    display: flex;   
+    justify-content: space-between;
+    align-items: center;
+    box-sizing: border-box;
+    padding: 20px 40px;
+    height: 90px;
+}
+
+nav ul {
+    display: flex;
+    justify-content: space-between;
+    list-style: none;
+    font-size: 1.35rem;
+    font-weight: 500;
+    cursor: pointer;      
+}
+
+nav ul li {
+    padding: 0 20px;    
+}
+
+nav ul li a {
+    text-decoration: none;
+    color: #757e83;
+}
+
+nav ul li a:hover {
+    color: var(--orange);
+}
+
+.active {
+    color: black;
+}
+
+#hamburger {
+    display: none;
```

Comment:
Things with display `none` should not be left behind, unless the display settings are toggled in some way to make them appear at least sometimes.